### DISCUSSION TITLE
Error in authentik log after setting up audiobookshelf with OIDC

### DISCUSSION THREAD

**Discussion by rama31244** [goauthentik/authentik#9643] [Q&A]
Hi
I tried to set up audiobookshelf with authentik in front, using OIDC, but when I launch the audiobookshelf domain, I get a 404 error. The authentik log shows the following error:

{"auth_via": "unauthenticated", "domain_url": "audiobookshelf.domain.com", "event": "/outpost.goauthentik.io/auth/traefik", "host": "audiobooks.domain.com", "level": "info", "logger": "authentik.asgi", "method": "GET", "pid": 53, "remote": "***.***.***.***", "request_id": "*******************", "runtime": 4, "schema_name": "public", "scheme": "https", "status": 404, "timestamp": "2024-05-08T10:59:22.739126", "user": "", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:125.0) Gecko/20100101 Firefox/125.0"} 

Can anyone help me work out why my audiobookshelf application isn't accessible when I try to protect it with authentik? My other applications that are protected with forward auth proxy providers work well. I also use traefik reverse proxy if this is relevant

**Comment by satrapes** [comment]
Hello, I am roughly in the same boat as you. I use SWAG from Linuxserver.io which is nginx based.
So one of the things that I tried is removing the proxy from the middle and audiobookshelf seems to work without a problem.
To do this, I connected to authentik with an IP. Copied the links from the provider into ABS. Added the ABS redirect URL in the provider so that they could find each other through their IP addresses.
When I added the proxy back into the mix by using domain names I get the 500 internal server error in ABS or the 404 not found from the logs as you mention above.
What I don't understand and I don't have the knowledge to find out is why does the proxy cause troubles. (I think I have followed authentik's configuration correctly)

**Comment by rama31244** [comment]
I actually ended up solving my problem, turns out I had forward auth middleware set up in Traefik that I forgot to remove when switching from proxy provider to OIDC. Don't know much about using a forward proxy like you are trying to achieve. Hopefully you work it out soon
